tests/integration/utils/utils.py

```diff
@@ -35,13 +35,26 @@ def compare_array(a, b, check_type=True):
     return True, None
 
 
-def check_array_method(fn, args, kwargs, print_msg, check_type=True):
+def check_array_method(
+    fn,
+    args,
+    kwargs,
+    print_msg,
+    check_type=True,
+    ndarray_func=False,
+    ndarray_np=[],
+):
     """
     Run np.func and num.func respectively and compare results
     """
 
-    a = getattr(np, fn)(*args, **kwargs)
-    b = getattr(num, fn)(*args, **kwargs)
+    if not ndarray_func:
+        a = getattr(np, fn)(*args, **kwargs)
+        b = getattr(num, fn)(*args, **kwargs)
+    else:
+        ndarray_num = num.array(ndarray_np)
+        a = getattr(ndarray_np, fn)(*args, **kwargs)
+        b = getattr(ndarray_num, fn)(*args, **kwargs)
```

Comment:
This conditional makes the helper mis-named since the other branch is testing a module functions, not array methods. But really, this should probably be split into two separate helpers `check_array_method` and `check_module_function`. Doing that should also help reduce the number of parameters (7 is officially "a lot")

Reply:
Done. Thanks.